go.mod

```diff
@@ -1,14 +1,27 @@
 module github.com/mendersoftware/mender-artifact
 
+go 1.14
```

Comment:
this should align with: https://github.com/mendersoftware/mender-artifact/blob/master/Dockerfile#L1 as it's built within docker... I'd suggesting removing it here, upgrading the base image to use a 1.14 based one and verifying the go mod upgrades in the image...

Reply:
Yes, sir!

Just to be clear that we are in sync here.

You are suggesting removing the `1.14` version in go.mod?

Or simply upgrading it to `1.14` in the Dockerfile?

Reply:
both 😄 - so that we at best have only place to set the go version, while the rest just picks it up... otherwise we could end up having some packages installed that require 1.14 and then fail because the image it's built in is just 1.11...

Reply:
I don't see how I can add this in one place though, unless I'm missing something obvious?

If not, I will upgrade the `Dockerfile` to `1.14`

Reply:
During the build (which takes place according to the `Dockerfile`) the environment `go.mod` will be executed in, has a go version set - so the setting in `go.mod` shouldn't be required, since the environment already dictates it.

Reply:
The structure of the `go.mod` file sort of requires this.

It is a 'minimum' tag for building this binary, sort of. Although I don't think it's much enforced yet. Older version will still try to build, and seems like they succeed mostly too.

Still, I can't remove that line.

Does that make sense?

Reply:
ah sorry so it's more like https://github.com/mapbox/mapbox-gl-js/blob/master/package.json#L13, then I just completely misunderstood that line... 🙈- having the version aligned with the `Dockerfile` should still be good to have I guess...

Reply:
Yes, seems exactly like that :)

Yes, did upgrade the go versions in the Dockerfiles to align :racehorse: